test/image-blueprints-bootc/layer2-source/group2/rhel95-bootc-source-ai-model-serving.containerfile

```diff
@@ -0,0 +1,235 @@
+#
+# Note: This requires setting the additional image stores path to a read-write
+# location on the file system. The images will still be treated as read-only by
+# the container subsystem.
+# See https://github.com/ostreedev/ostree-rs-ext/issues/654
+COPY --chmod=755 ./bootc-images/microshift-imagestore-config.sh /usr/bin/microshift-imagestore-config
+COPY --chmod=644 ./bootc-images/microshift-imagestore-config.service /etc/systemd/system/microshift-imagestore-config.service
+RUN systemctl enable microshift-imagestore-config.service
+
+# Test data
+# hadolint ignore=DL3059
+RUN mkdir /etc/microshift/manifests.d/10-ai-model-serving-test/
+
+# hadolint ignore=DL3059
+RUN cat <<EOF > /etc/microshift/manifests.d/10-ai-model-serving-test/kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- resources.yaml
+EOF
+
+# hadolint ignore=DL3059
+RUN cat <<EOF > /etc/microshift/manifests.d/10-ai-model-serving-test/resources.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: test-ai
+---
+# ServingRuntime based on what is shipped in the redhat-ods-applications namespace.
+# Needs to be copied to workload namespace (test-ai).
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  annotations:
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+    openshift.io/display-name: OpenVINO Model Server
+  labels:
+    opendatahub.io/dashboard: "true"
+  name: kserve-ovms
+  namespace: test-ai
+spec:
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "8888"
+  containers:
+  - args:
+    - --model_name={{`{{.Name}}`}}
+    - --port=8001
+    - --rest_port=8888
+    - --model_path=/mnt/models
+    - --file_system_poll_wait_seconds=0
+    - --grpc_bind_address=0.0.0.0
+    - --rest_bind_address=0.0.0.0
+    - --target_device=AUTO
+    - --metrics_enable
+    image:  {{ .Env.SOURCE_IMAGES_AIMS_OVMS }}
+    name: kserve-container
+    ports:
+    - containerPort: 8888
+      protocol: TCP
+  multiModel: false
+  protocolVersions:
+  - v2
+  - grpc-v2
+  supportedModelFormats:
+  - autoSelect: true
+    name: openvino_ir
+    version: opset13
+  - name: onnx
+    version: "1"
+  - autoSelect: true
+    name: tensorflow
+    version: "1"
+  - autoSelect: true
+    name: tensorflow
+    version: "2"
+  - autoSelect: true
+    name: paddle
+    version: "2"
+  - autoSelect: true
+    name: pytorch
+    version: "2"
+---
+# Definition of InferenceService with a model packages in form of OCI image.
+# Features extra argument for the model server (--layout), so the data layout
+# format expected by the model server matches what we send during testing.
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: openvino-resnet
+  namespace: test-ai
+spec:
+  predictor:
+    model:
+      protocolVersion: v2
+      modelFormat:
+        name: openvino_ir
+      storageUri: "oci://quay.io/pmatusza/microshift-testing:resnet"
```

Comment:
I want to test it, but we can discuss how to name that image. I think there will be another image with model for aws+nvidia test (some granite).
So perhaps (I tweaked @agullon's idea): 
- `quay.io/microshift/ai-testing-model:ovms-resnet50`
- `quay.io/microshift/ai-testing-model:vllm-granite-8b-code-base-4k`


ovms = OpenVino Model Serving